packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -147,8 +159,36 @@ class RenderSliverFixedExtentList extends RenderSliverFixedExtentBoxAdaptor {
 class RenderSliverFill extends RenderSliverFixedExtentBoxAdaptor {
   RenderSliverFill({
     @required RenderSliverBoxChildManager childManager,
-  }) : super(childManager: childManager);
+    double peek: 0.0,
+  }) : _peek = peek, super(childManager: childManager) {
+    assert(peek != null);
+  }
 
   @override
-  double get itemExtent => constraints.viewportMainAxisExtent;
+  double get itemExtent => constraints.viewportMainAxisExtent - 2.0 * peek;
+
+  double get peek => _peek;
+  double _peek;
+  set peek (double newValue) {
+    assert(newValue != null);
```

Comment:
Oh, I see, peek values are absolute pixels. This seems surprising, I would have expected it to be a fraction of the viewportMainAxisExtent.